### DISCUSSION TITLE
Italics and bold

### DISCUSSION THREAD

**Discussion by bdwhitebd** [sffc/card-creatr-studio#84] [Q&A]
Is there a way to stylize text within the textWrap function? I have card text written in a multiline description, but I was hoping to be able to use bold and/or italics on certain words within the description. HTML tags within the multiline textwrap are just rendered as plain text.

**Comment by sffc** [comment]
Good question. Short answer is "not technically supported right now", *but* a workaround is to use stylized characters (in a font that supports them). Here are some web sites that can help with this:

https://yaytext.com/bold-italic/

https://lingojam.com/BoldTextGenerator

https://texttools.org/bold-text-generator
